src/main/java/com/gengo/client/GengoClient.java

```diff
@@ -515,8 +682,82 @@ public JSONObject determineTranslationCost(Payloads jobs) throws GengoException
      */
     public JSONObject getOrderJobs(int orderId) throws GengoException
     {
-        String url = baseUrl + "translate/order/";
+        String url = getBaseUrl() + "translate/order/";
+        url += orderId;
+        return call(url, HttpMethod.GET);
+    }
+
+    /**
+     * Delete translation job.
+     * @param orderId
+     * @return
+     * @throws GengoException
+     */
+    public JSONObject deleteTranslationOrder(Integer orderId) throws GengoException
+    {
+    	String url = getBaseUrl() + "translate/order/";
+    	url += orderId;
+    	return call(url, HttpMethod.DELETE);
+    }
+
+    /**
+     * Get entire comments for the specific order.
+     * @param orderId
+     * @return
+     * @throws GengoException
+     */
+    public JSONObject getOrderComments(Integer orderId) throws GengoException
+    {
+        String url = getBaseUrl() + "translate/order/";
         url += orderId;
+        url += "/comments";
+        return call(url, HttpMethod.GET);
+    }
+
+    /**
+     * Register comment for the specific order.
+     * @param orderId
+     * @param comment
+     * @return
+     * @throws GengoException
+     */
+    public JSONObject postOrderComment(Integer orderId, String comment) throws GengoException
```

Comment:
In this case, I agree with that `int` is reasonable so I'll update here. Thanks.